tools/shared.py

```diff
@@ -931,8 +935,10 @@ def get_llvm_target():
 
 if LLVM_TARGET == WASM_TARGET:
   # wasm target does not automatically define emscripten stuff, so do it here.
-  COMPILER_OPTS = COMPILER_OPTS + ['-DEMSCRIPTEN',
-                                   '-D__EMSCRIPTEN__']
+  COMPILER_OPTS = COMPILER_OPTS + ['-D__EMSCRIPTEN__']
```

Comment:
looks like this setting of the emscripten define is duplicated in `emcc.py`? if we need it in both (but do we?), perhaps we could refactor it into a shared method?